### ISSUE TITLE
alternate approach to Linux: standalone GUI which consumes config files

### ISSUE THREAD

**Issue by restena-sw** [GEANT/CAT#7]
This actually exists, but is not the default. Since it slightly complicates the user experience, it's not an easy decision to make the switch.

**Comment by pikachuprof** [comment]
just leaving a comment - it is to hard for most of our users on Android to use this two steps so even if we assume Linux users are a bit more "advanced" I don't think this would be a good idea. A advanced user could just configure /etc/NetworManager/system-connections/eduroam himself anyway so...

**Comment by ikreb7** [comment]
I would agree with @pikachuprof that more than one file is too complicated. But we could offer a binary file where more files are included.

**Comment by rugk** [comment]
Why re-invent the wheel? I don't like this suggestion, so IMHO adhere to standards/existing GUIs and explicitly _do not_ implement your own thing. This is just not needed and unnecessary work.
